test/exercises/boundedBuffer/solutions/boundedBuffer-atomic.chpl

```diff
@@ -26,7 +26,7 @@ config const numProducers = 5,     // the number of producers to create
 
 proc main() {
   // a shared bounded buffer with the requested capacity
-  var buffer = new BoundedBuffer(capacity=bufferSize);
+  var buffer = new unmanaged BoundedBuffer(capacity=bufferSize);
```

Comment:
I imagine longer term the bounded buffer exercises should use managed classes since they're used in tutorials (at least I think they are)

Reply:
It also seems like the main boundedBuffer.chpl is getting out of sync with the solutions ones -- e.g. this is using an `unmanaged RandomStream` and boundedBuffer.chpl is using an `owned RandomStream`

Reply:
Actually AFAIK they're not currently used in tutorials. I'd expect we'd have another look before doing such a tutorial. (But other people might run tutorials off of them...)